drivers/mpathcount.py

```diff
@@ -246,7 +253,9 @@ def add(key, val):
             if srtype in supported:
                 devconfig = record["device_config"]
                 sm_config = session.xenapi.SR.get_sm_config(SR)
-                check_devconfig(devconfig, sm_config, config, remove, add)
+                check_devconfig(devconfig, sm_config, config, remove, add, mpath_status)
+        util.atomicFileWrite(MPATH_FILE_NAME, MPATHS_DIR, json.dumps(mpath_status))
```

Comment:
These two lines should be indented to match the check_devconfig above

Reply:
This file `MPATH_FILE_NAME` is created for record the multi-path status per host, and it dumps the status of all PBD's paths. therefore, I think these two lines may not be indented to match `check_devconfig ` ?